lua/barbar/api.lua

```diff
@@ -59,9 +60,15 @@ local function notify_buffer_not_found(buffer_number)
 end
 
 --- Forwards some `order_func` after ensuring that all buffers sorted in the order of pinned first.
---- @param order_func fun(bufnr_a: integer, bufnr_b: integer): boolean accepts `(integer, integer)` params.
+--- @param order_func fun(bufnr_a: integer, bufnr_b: integer, to_sort_case: fun(s: string): string): boolean accepts `(integer, integer)` params.
 --- @return fun(bufnr_a: integer, bufnr_b: integer): boolean
 local function with_pin_order(order_func)
+  local to_sort_case = config.options.sort.ignore_case and function(s)
+    return tolower(s)
+  end or function(s)
```

Comment:
Isn't `function(s) return tolower(s) end` equivalent to just `tolower`?

Reply:
You're totally right— good catch!